src/index.html

```diff
@@ -7,12 +7,61 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="container"
+      data-qa="card"
+    >
+      <img
+        class="container__image"
+        src="images/imac.jpeg"
+        alt="Apple Device(imac)"
+      />
+      <div class="container__product-info">
+        <h3 class="product-info-title">
```

Comment:
Class names should be written using the BEM methodology. The class `product-info-title` should be `container__product-info-title` to maintain consistency and clarity in the naming convention.